lib/haml_lint/linter/alt_text.rb

```diff
@@ -0,0 +1,11 @@
+module HamlLint
+  class Linter::AltText < Linter
+    include LinterRegistry
+
+    def visit_tag(node)
+      if node.tag_name == 'img' && !node.has_hash_attribute?(:alt)
```

Comment:
The name is fine. I guess I was just expecting a list of different tags, so code more along the lines of:

```ruby
TAGS_REQUIRING_ALT = %w[img ...]

...

  if TAGS_REQUIRING_ALT.include?(node.tag_name) && ...
    ...
  end
```